docs/public/specifications/tests/Nameserver-TP/nameserver15.md

```diff
@@ -0,0 +1,119 @@
+# NAMESERVER15: Checking for software version
+
+## Test case identifier
+
+**NAMESERVER15**
+
+## Table of contents
+
+* [Objective](#objective)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
+* [Terminology](#terminology)
+
+## Objective
+
+This Test Case verifies if a name server responds to TXT queries in the CHAOS class, specifically
+about its software version as it may sometimes be desirable not to reveal that information.
+
+A description of DNS classes can be found in [RFC2929], section 3.2.
+
+## Scope
+
+It is assumed that *Child Zone* is also tested and reported by [Connectivity01].
+This Test Case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+Message Tag                 | Level   | Arguments                     | Message ID for message tag
+:---------------------------|:--------|:------------------------------|:---------------------------------------------------------------------------------------
+N15_BIND_VERSION            | INFO    | ns_ip_list, string            | Name server(s) "{ns_ip_list}" respond(s) to CHAOS TXT query "version.bind" with value "{string}".
```

Comment:
The "ns_ip_list" can be a long list. I always put that at the end or else reading may break.

Since the value can be anything, isn't "with string" better?

"string" is not a defined argument name. Should it be "string" or "bind_version"? Please update the argument list.

Is "CHAOS TXT" relevant here? Or should it rather be "name server version query"?

Reply:
I’d move the `{ns_ip_list}` at the end, but apart from that, the message ID looks fine to me.

Reply:
There is also "version.server" to query for, and then a more generic msgid is reasonable.